Source/WebCore/page/LocalDOMWindow.h

```diff
@@ -283,6 +283,8 @@ class LocalDOMWindow final
 
     using EventTarget::dispatchEvent;
     void dispatchEvent(Event&, EventTarget*);
+    bool hasDispatchedInputEvent() const { return m_hasDispatchedInputEvent; }
```

Comment:
I think you should add a comment pointing to https://github.com/w3c/event-timing/issues/158, so that readers don't get the wrong understanding about what this is. Also group it with performance API functions, not right next to `dispatchEvent` which reinforces the misunderstanding.